Adafruit_MQTT.cpp

```diff
@@ -265,17 +265,19 @@ uint16_t Adafruit_MQTT::readFullPacket(uint8_t *buffer, uint16_t maxsize,
   } while (encodedByte & 0x80);
 
   DEBUG_PRINT(F("Packet Length:\t"));
-  DEBUG_PRINTLN(value);
+  DEBUG_PRINTLN(fixedHeaderLength + remainingLength);
 
-  if (value > (maxsize - (pbuff - buffer) - 1)) {
+  uint16_t fixedHeaderLength = pbuff - buffer;
```

Comment:
NIT: make it const?
```
const uint16_t fixedHeaderLength = pbuff - buffer;
```

Reply:
While semantically correct, since the `fixedHeaderLength` doesn't change afterwards, `const` is currently only used for parameters and return values. It doesn't help the compiler or consumer of the API much either.